### PR TITLE
set log level flag

### PR DESCRIPTION
**Pull request by ryancouto**
Allows binaries to take a flag to set their log level. This way we can disable debug for noisy logs in travisCI, etc.

### CODE REVIEW

File: binaries/setup-cloud-scoot/main.go [hunk 1]
```diff
@@ -31,13 +38,13 @@ func main() {
 
 	workersCfg := &setup.WorkerConfig{Count: *workersFlag, RepoDir: *repoDir}
 	sched := map[string]setup.SchedulerStrategy{
-		"local.memory": setup.NewLocalMemory(workersCfg, builder, cmds),
-		"local.local":  setup.NewLocalLocal(workersCfg, builder, cmds),
+		"local.memory": setup.NewLocalMemory(workersCfg, builder, cmds, level),
```
Comment (jasonbs10) [inline binaries/setup-cloud-scoot/main.go]: Can you move level into the WorkerConfig struct instead? Same for scheduler and apiserver.

File: scootapi/setup/api.go [hunk 2]
```diff
@@ -26,17 +26,19 @@ type ApiConfig struct {
...
-func NewLocalApiStrategy(apiCfg *ApiConfig, builder Builder, cmds *Cmds) *LocalApiStrategy {
+func NewLocalApiStrategy(apiCfg *ApiConfig, builder Builder, cmds *Cmds, level log.Level) *LocalApiStrategy {
 	return &LocalApiStrategy{
-		apiCfg:  apiCfg,
-		builder: builder,
-		cmds:    cmds,
+		apiCfg:   apiCfg,
+		builder:  builder,
+		cmds:     cmds,
+		logLevel: level,
```
Comment (jasonbs10) [inline scootapi/setup/api.go]: For all of these strategies, can you move "level" into the higher level *Config struct and then access it as localApiStrategy.apiCfg.level?

File: tests/testhelpers/clusterHelpers.go [hunk 3]
```diff
@@ -22,10 +22,10 @@ func CreateLocalTestCluster() (*setup.Cmds, error) {
 	builder := setup.NewGoBuilder(clusterCmds)
 	go func() {
 		sched := map[string]setup.SchedulerStrategy{
-			"local.local": setup.NewLocalLocal(nil, builder, clusterCmds),
+			"local.local": setup.NewLocalLocal(nil, builder, clusterCmds, log.DebugLevel),
```
Comment (jasonbs10) [inline tests/testhelpers/clusterHelpers.go]: Is DebugLevel the right choice here? I don't actually know, but we for sure don't want debug in TravisCI.

File: binaries/setup-cloud-scoot/main.go (hunk 1 above)
Comment (ryancouto) [inline binaries/setup-cloud-scoot/main.go]: There isn't a SchedulerConfig, so the scheduler's log level will be set through the WorkerConfig.

File: tests/testhelpers/clusterHelpers.go (hunk 3 above)
Comment (ryancouto) [inline tests/testhelpers/clusterHelpers.go]: No, you're right. Info is a better default.

**Comment by jasonbs10** [comment]
lgtm

**Pull request merged**